src/reportingApi.ts

```diff
@@ -26,4 +27,30 @@ export const ReportingApi = {
     sendEventToReporter(EVENTS.SET_DESCRIPTION, description, suite),
   setTestCaseId: (testCaseId: string, suite?: string) =>
     sendEventToReporter(EVENTS.SET_TEST_CASE_ID, testCaseId, suite),
+  setStatusPassed: (suite?: string) =>
```

Comment:
Please provide methods like `setStatus` and` setLaunchStatus` where user can pass status by self